lib/shelley/test/data/cardano-node-shelley/genesis.yaml

```diff
@@ -34,10 +34,10 @@ initialFunds: {}
 maxLovelaceSupply: 45000000000000000
 networkMagic: 1
 networkId: Mainnet
-epochLength: 1215
+epochLength: 100
 staking:
 slotsPerKESPeriod: 86400
-slotLength: 8
+slotLength: 0.2
```

Comment:
Wow, that's a short slot. So if the tests get paused for 100x0.2=20secs (for whatever reason), they will not be able to continue.

Reply:
I've tweaked according to what the ledger / consensus teams are using for testing as well; it seems fairly okay though. I don't get why the tests wouldn't be able to continue on if they pause for 20s :thinking: ?

Reply:
The epoch length is 20s and I _think_ if an entire epoch's worth of slots are missed then it will not be able to elect slot leaders for the next epoch. But it should be ok anyway. Nobody's going to press ctrl-z during integration tests and expect them to pass.

Reply:
The cluster is made of one BFT leader + 3 pools though, so there's always the BFT leader capable of producing blocks. I am not sure how that influence the slot leader election however :thinking:

Reply:
> Nobody's going to press ctrl-z during integration tests and expect them to pass.

I guess that's also a fair assumption :)